crates/bevy_gltf/src/loader.rs

```diff
@@ -978,4 +983,50 @@ mod test {
         assert_eq!(result[0].0, "l2");
         assert_eq!(result[0].1.children.len(), 0);
     }
+
+    use super::Gltf;
+    use bevy_app::{App, AppExit, EventWriter};
+    use bevy_asset::{AssetPlugin, AssetServer, Assets, Handle};
+    use bevy_core::CorePlugin;
+    use bevy_ecs::system::{Commands, Res};
+    use bevy_scene::Scene;
+
+    #[test]
+    fn test_scene_to_nodes() {
+        App::new()
+            .add_plugin(bevy_log::LogPlugin::default())
+            .add_plugin(CorePlugin::default())
+            .add_plugin(AssetPlugin::default())
+            .add_plugin(bevy_scene::ScenePlugin::default())
+            .add_plugin(GltfPlugin::default())
+            .add_startup_system(setup)
+            .add_system(spawn_gltf_objects)
+            .run();
+    }
+
+    fn setup(mut commands: Commands, assets: Res<AssetServer>) {
+        let handle: Handle<Gltf> = assets.load("models/FlightHelmet/FlightHelmet.gltf");
```

Comment:
This path doesn't work for loading the assets. However this doesn't cause an error and the test just exits. Which I don't want but don't know how to fix.

Reply:
I fixed this by copying over the asset.
This is not good, but I didn't know what else to do, as cargo controls the relevant environment variable.
See [the docs](https://docs.rs/bevy_internal/0.6.0/bevy_internal/asset/prelude/struct.AssetServer.html#method.load)